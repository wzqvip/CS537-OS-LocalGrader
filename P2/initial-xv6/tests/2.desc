opening a file, and then calling the system call with that file descriptor
