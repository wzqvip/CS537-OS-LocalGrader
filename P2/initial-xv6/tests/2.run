~cs537-1/tests/P2/tester/run-xv6-command.exp CPUS=1 Makefile.test test_2 | grep XV6_TEST_OUTPUT
