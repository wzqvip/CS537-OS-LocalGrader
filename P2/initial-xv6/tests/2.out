XV6_TEST_OUTPUT Open filename: README
